### PR TITLE
finished issue #28

### PR DESCRIPTION
**Pull request by badboy-huaqiao**
about adding Makefile targets to build docker image , i have finished it.
Signed-off-by: Huaqiao Zhang <huaqiaoz@vmware.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,4 +38,9 @@ prepare:
...
 
 run:
-	cd bin && ./edgex-ui-go-launch.sh
\ No newline at end of file
+	cd bin && ./edgex-ui-go-launch.sh
+
+docker: $(DOCKERS)
+
+docker_edgex_ui_go:
+	docker build -f docker-file/Dockerfile -t edgexfoundry/docker-edgex-ui-go:$(VERSION) .
```
Comment (JPWKU) [inline Makefile]: We also need to add a label flag here that puts in the GIT_SHA.

Comment (badboy-huaqiao) [reply Makefile]: ok, i will fix it

Comment (JPWKU) [reply Makefile]: Thanks @badboy-huaqiao it should be a really easy fix.
Add a line like this https://github.com/edgexfoundry/edgex-go/blob/master/Makefile#L25
and then add a flag in the docker build command string like this
https://github.com/edgexfoundry/edgex-go/blob/master/Makefile#L81

With this we can always inspect a container and see the exact code it was built from.

Comment (badboy-huaqiao) [reply Makefile]: @JPWKU thanks for your reminding，it's done now ! if any need to be modified, please leave msg here, thanks.

**Comment by JPWKU** [comment]
@badboy-huaqiao We need this implemented for delhi branch as well.  I think you might be able to cherry pick this change into another branch to PR against delhi.  Also the master branch VERSION file will need to be the version for future release of edinburgh and delhi branch VERSION for this Friday's release.

**Pull request merged**